### ISSUE TITLE
isPathInside is reversed?

### ISSUE THREAD

**Issue by LoganDark** [NeekSandhu/path-fx#4]
The docs say the first parameter is `containingPath` and the second parameter is `path`, so I assumed that you pass the directory as the first parameter and something inside that directory as the second.

Nope. They are completely reversed.

![image](https://user-images.githubusercontent.com/4723091/135790408-e834eb89-8ce1-481e-9de6-0f25a9f2106d.png)

You should absolutely swap the names or change them to something else.

**Comment by zikaari** [comment]
Interesting. I'm gonna have to investigate this carefully. Will get back to you over the weekend.

**Comment by LoganDark** [comment]
The only thing that can change is documentation, or else every application using this API will break.

**Comment by zikaari** [comment]
Yeah, that's the part I want to carefully gauge. I want to see what circumstances even led to such a disparity.
